### ISSUE TITLE
esgdrs failure on non-ascii character in netCDF attribute

### ISSUE THREAD

**Issue by alaniwi** [ESGF/esgf-prepare#55]
With a netCDF file containing this global attribute:
```
                :grid = "regular 1/2° lat-lon grid" ;
```
Trying to run `esgdrs todo` or `esgdrs upgrade` (from esgprep v2.9.4 2018-10-12) gives:
```
:: SKIPPED :: /tmp/data/wtd_Emon_CNRM-CM6-1_amip_r1i1p1f2_gn_197901-201412.nc
Traceback (most recent call last):
  File "/path/to/esgprep/drs/main.py", line 55, in process
    set_values=pctx.set_values)
  File "/path/to/esgprep/drs/handler.py", line 91, in load_attributes
    self.attributes[attr] = str(nc.getncattr(attr)).split()[0]
UnicodeEncodeError: 'ascii' codec can't encode character u'\ufffd' in position 11: ordinal not in range(128)
Number of file(s) scanned: 0
Number of error(s): 0
```

Steps to reproduce:
* `ncatted -a grid,global,o,c,"regular 1/2° lat-lon grid"  <filename>`
* run esgdrs

If I use `ncatted` to remove this character, then `esgdrs` will run successfully.

I suspect that use of `unicode` in place of `str` may solve it, but I have not tested it.

I have not tested the latest version, but the `str` call which raises the exception is still present in the code (now at line 90):

https://github.com/ESGF/esgf-prepare/blob/e3a749b4d7f6034e5330258dbcef89a79df689da/esgprep/drs/handler.py#L90

**Comment by glevava** [comment]
Thanks @alaniwi for catching this issue. I will solve it (probably using unicode indeed) in the next week.

**Comment by glevava** [comment]
Solved in 2.9.5 release.
